articles/quickstarts/index.md

```diff
@@ -19,17 +19,144 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
+        dotnet iqsharp install
+    ```
 
-[Develop with Q# and .NET](xref:microsoft.quantum.install.cs) - Combine C#, F#, or VB.NET with Q# to create a .NET host program that calls Q# operations.
+    > [!NOTE]
+    > - If you get an error during the `dotnet iqsharp install` step, open a new terminal window and try again.
+    > - If this still doesn't work, locate the installed `dotnet-iqsharp` tool (on Windows, `dotnet-iqsharp.exe`) and run: <br>`<path to dotnet-iqsharp> install --user --path-to-tool=<path to dotnet-iqsharp>`,<br> where `<path to dotnet-iqsharp>` should be replaced by the absolute path to the `dotnet-iqsharp` tool in your file system.
+     Typically this will be under `.dotnet/tools` in your user profile folder.
+    > - For additional debugging help, see https://natemcmaster.com/blog/2018/02/02/dotnet-global-tool/.
+
+*** 
+
+You now have the IQ# kernel for Jupyter, which provides the core functionality for compiling and running Q# operations from Q# Jupyter Notebooks.
+
+## Use the QDK Online
+
+You can also develop Q# code without installing anything locally with these options:
+
+|Resource|Advantages|Limitations|
+|---|---|---|
+|**Visual Studio Codespaces**|A rich online development environment  |Requires an Azure subscription and plan |
+|**Visual Studio Code GitHub Codespaces (beta)**|Online Visual Studio Code experience |Currently in beta, request early access at https://github.com/features/codespaces |
+|**MyBinder** | Free online notebook experience |No persistence |
+
+### Using Visual Studio Codespaces
+
+To use Visual Studio Codespaces for Q# development:
+
+1. Create an [Azure account](https://azure.microsoft.com/free/).
+2. Create a Codespaces environment, following the [quickstart guide](https://docs.microsoft.com/visualstudio/codespaces/quickstarts/browser). When creating your codespace, we recommend using *microsoft/quantum* in the **Git Repository** field to load QDK-specific settings.
+3. You can now launch your new environment and start developing in the browser via the [VS Codespaces Cloud IDE](https://online.visualstudio.com/environments). Alternatively, it is possible to use your local installation of VS Code and use Codespaces as a [remote environment](https://docs.microsoft.com/visualstudio/online/how-to/vscode).
+
+### Using Binder
+
+You can use Binder to run your Jupyter Notebooks online.
+
+To configure Binder automatically and experiment with the QDK samples:
+
+- Open a browser and run https://aka.ms/try-qsharp.
+
+To configure Binder manually:
+
+1. Go to [mybinder.org](https://mybinder.org).
+1. Fill out the fields, using *microsoft/quantum* as the GitHub URL.
+
+![MyBinder form](~/media/mybinder.png)
+
+## Use the QDK with Docker
+
+You can use our QDK Docker image in your local Docker installation or in the cloud via any service that supports Docker images, such as ACI.
+
+You can download the IQ# Docker image from https://github.com/microsoft/iqsharp/#using-iq-as-a-container.
+
+> [!NOTE]
```

Comment:
Not sure this part requires a special NOTE. I would just add it as another sentence in the same format.

Reply:
Fixed